sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/BeamFnControlClient.java

```diff
@@ -137,6 +137,7 @@ public void processInstructionRequests(Executor executor)
       executor.execute(
           () -> sendInstructionResponse(delegateOnInstructionRequestType(currentRequest)));
     }
+    Thread.sleep(50);
```

Comment:
<!--new_thread; commit:5300be61ccac7a006ec7f2b8f3824cddb4dc75e8; resolved:0-->
Comment as to why?

Reply:
Leftover debugging. (There used to be some logging and flushing here as well.) Removed.